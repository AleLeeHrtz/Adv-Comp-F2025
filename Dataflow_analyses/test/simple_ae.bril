@main {
  a: int = const 1;
  b: int = const 2;
  c: int = add a b;
  d: int = mul a b;
  a: int = const 5;
  e: int = add a b;
}
