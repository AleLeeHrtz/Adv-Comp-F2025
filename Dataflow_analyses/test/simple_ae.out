=== Available Expressions Analysis ===
entry:
  in:  âˆ…
  out: ('add', 'a', 'b')
